include/Core/Controller.php

```diff
@@ -161,6 +162,31 @@ public function load_taxonomy_term() {
 		die;
 	}
 
+	/**
+	 * Ajax call back function for getting taxonomies meta to load select2 options.
+	 *
+	 * @since 6.0.0
+	 */
+	public function load_taxonomy_term_meta() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [load_taxonomy_term_meta accesses the super-global variable $_GET.](https://app.codacy.com/app/BulkWP/bulk-delete/pullRequest?prid=1939930)